.gitignore

```diff
@@ -26,6 +26,7 @@ packages/amplify-e2e-tests/amplify-e2e-reports
 packages/*/node_modules
 packages/*/package-lock.json
 **/.env
+!packages/amplify-e2e-tests/projects-templates/create-react-app-auth-amplify/.env
```

Comment:
Do we need this if we completely delete the folder?

Reply:
Nope. This folder is used to act as an template projects that is used for tests. I created a new initAuthProject(), which will init project with content for this folder